src/features/content-answers/ContentAnswersModalContent.tsx

```diff
@@ -13,28 +13,60 @@ import { User } from '../../../common/types/core';
 import './ContentAnswersModalContent.scss';
 
 type Props = {
-    currentUser: User;
+    currentUser?: User;
     fileName: string;
     isLoading: boolean;
     questions: QuestionType[];
 };
 
 const ContentAnswersModalContent = ({ currentUser, fileName, isLoading, questions }: Props) => {
+    const messagesEndRef = useRef<HTMLDivElement | null>(null);
+
+    const handleScrollToBottom = useCallback(behavior => {
+        if (messagesEndRef.current && messagesEndRef.current.scrollIntoView) {
+            messagesEndRef.current.scrollIntoView({ behavior });
+        }
+    }, []);
+
+    const throttledScrollToBottom = throttle(handleScrollToBottom, 1000);
```

Comment:
is this 1 second? sounds sort of long, is that to fix the scrolling issue?

Reply:
actually just got an update that this throttling is for streaming mode, so we could remove this logic